cocotb/ipython_support.py

```diff
@@ -61,7 +61,7 @@ async def embed(user_ns: dict = {}):
     c = load_default_config()
     c.TerminalInteractiveShell.loop_runner = lambda x: _runner(shell, x)
     c.TerminalInteractiveShell.autoawait = True
-
+    c.HistoryAccessor.connection_options = {'check_same_thread' : False }
```

Comment:
I assume black wants this formatted as:
```suggestion
    c.HistoryAccessor.connection_options = {"check_same_thread": False}
```
(can't check right now).